packages/prisma/seed-app-store.ts

```diff
@@ -316,6 +316,12 @@ export default async function main() {
       client_secret: process.env.ZOHOCRM_CLIENT_SECRET,
     });
   }
+  if (process.env.ZOHOCALENDAR_CLIENT_ID && process.env.ZOHOCALENDAR_CLIENT_SECRET) {
+    await createApp("zohocalendar", "zohocalendar", ["calendar"], "zohocalendar_calendar", {
+      client_id: process.env.ZOHOCALENDAR_CLIENT_ID,
+      client_secret: process.env.ZOHOCALENDAR_CLIENT_SECRET,
+    });
+  }
```

Comment:
We don't need to update this file any more. This is just legacy now. An App is seeded when someone goes to settings/admin/apps and enable the app.